src/Add_remove.js

```diff
@@ -0,0 +1,28 @@
+const TodoLoad = (todos) => {
+  todos = JSON.parse(localStorage.getItem('todos')) || [];
+  for (let i = 0; i < todos.length; i += 1) {
+    todos[i].index = i + 1;
+  }
+  return todos;
+};
+
+class Todo {
+  constructor(description, index = 0, completed = false) {
+    this.index = index;
+    this.description = description;
+    this.completed = completed;
+  }
+
+  addTodo = (task, todos) => {
+    todos.push(task);
+    localStorage.setItem('todos', JSON.stringify(todos));
+  }
+
```

Comment:
- [ ] At the moment, when a to-do task is saved to localStorage, the `index `property is always set to `0`. This is not the expected behaviour. The `index` property for each to-do item should be set to the value of the new array length. Please implement this requirement 🙏 
Kindly see the bug below 👇 
![image](https://user-images.githubusercontent.com/46839548/184395922-e9c0740f-65bd-4195-939b-571e8d9db374.png)